collections/jobs/Tasks.ts

```diff
@@ -0,0 +1,49 @@
+import type { CollectionConfig } from 'payload'
+
+export const Tasks: CollectionConfig = {
+  slug: 'tasks',
+  admin: {
+    group: 'Jobs',
+    useAsTitle: 'title',
+  },
+  fields: [
+    {
+      name: 'title',
+      type: 'text',
+      required: true,
+    },
+    {
+      name: 'description',
+      type: 'textarea',
+    },
+    {
+      name: 'parent',
+      type: 'relationship',
+      relationTo: 'tasks' as any,
```

Comment:
whenever you see typescript errors for new collections, you need to run `pnpm generate:types` ... do that after adding new collections to resolve the typescript errors, and then please remove the `as any`.